### PR TITLE
installing the perl-bioperl-core to this tool's conda env made the tool work

### PR DESCRIPTION
**Pull request by bwlang**
before installing that  got this error :
```
Can't locate Bio/Root/RootI.pm in @INC (you may need to install the Bio::Root::RootI module) (@INC contains: blib/lib /home/galaxy/bioperl-live /mnt/galaxy/data/galaxy/tool-deps/_conda/envs/__perl-bioperl@1.6.924/lib/site_perl/5.26.2/x86_64-linux-thread-
```
not sure that's the best way, it installs tons of dependencies. but it does work with this extra package installed.

FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/bioperl/macros.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
     <xml name="requirements">
         <requirements>
             <requirement type="package" version="1.6.924">perl-bioperl</requirement>
+            <requirement type="package" version="1.007002">perl-bioperl-core</requirements>
```
Comment (mvdbeek) [inline tools/bioperl/macros.xml:5]:
```suggestion
            <requirement type="package" version="1.007002">perl-bioperl-core</requirement>
```

But this should be part of perl-bioperl. It is definitely part of the 1.7.2 recipe, and the scheduled tests also passed (https://github.com/galaxyproject/tools-iuc/actions/runs/107632312 -> All tool test results). Any chance that an old build of `1.6.924` got installed to your environment -- can you try installing `perl-bioperl=1.6.924-7` into your environment ?

**Comment by bwlang** [comment]
here is what was installed before i started messing around:
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       0_gnu    conda-forge
libgcc                    7.2.0                h69d50b8_2    conda-forge
libgcc-ng                 9.2.0                h24d8f2e_2    conda-forge
libgomp                   9.2.0                h24d8f2e_2    conda-forge
libstdcxx-ng              9.2.0                hdf63c60_2    conda-forge
perl                      5.26.2            h516909a_1006    conda-forge
perl-bioperl              1.6.924                       4    bioconda
perl-threaded             5.26.0                        0    bioconda
perl-yaml                 1.29                    pl526_0    bioconda


looks like version 4.

i didn't do anything special - just installed the tool so I don't know why it picked version 4... 
should I update the dependency to require 1.6.924-7 instead?

**Comment by mvdbeek** [comment]
> should I update the dependency to require 1.6.924-7 instead?

That should happen on new installs already, we don't want to be that precise in our requirements. I would assume your Conda resolver decided to optimize the installation to accomodate packages already present in one of your environemnts ... that is unfortunate. What version of Conda is this ?

**Comment by bwlang** [comment]
this was a fresh install from a day or 2 ago...
conda --version
conda 4.8.2

**Comment by mvdbeek** [comment]
How about trying with 1.72 as a compromise. We have the docker images in https://quay.io/repository/biocontainers/perl-bioperl?tag=latest&tab=tags that we know are working and for which we could provide ennironment.yaml files retrospectively. I have wanted to implement this for some time now, I think it would really take some edges of these Conda environemnts that degrade over time.

File: tools/bioperl/bp_genbank2gff3.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<tool id="bp_genbank2gff3" name="Genbank to GFF3" version="1.0">
+<tool id="bp_genbank2gff3" name="Genbank to GFF3" version="1.0a">
```
Comment (mvdbeek) [inline tools/bioperl/bp_genbank2gff3.xml:1]:
```suggestion
<tool id="bp_genbank2gff3" name="Genbank to GFF3" version="1.1">
```

**Comment by mvdbeek** [comment]
Thanks @bwlang!

**Pull request merged**